libdevcore/CommonData.h

```diff
@@ -140,6 +140,12 @@ inline bytes toCompactBigEndian(uint8_t _val, unsigned _min = 0)
 	return (_min || _val) ? bytes{ _val } : bytes{};
 }
 
+/// Workarounds shift left bug in boost <1.65.1.
```

Comment:
Maybe link the issue or at least mention the issue number?

Reply:
https://github.com/boostorg/multiprecision/issues/31

Reply:
I meant in the commend ^^ I mean that's where people will see it first and will be curious right?